resources/js/Layouts/AuthLayout.vue

```diff
@@ -0,0 +1,28 @@
+<script setup lang="ts">
+defineSlots<{
+  header: () => any
+  form: () => any
+  footer: () => any
+}>()
+</script>
```

Comment:
I don't think you need this extra code with `defineSlots`. It will work great without it.


```suggestion
```